fluss-common/src/main/java/com/alibaba/fluss/memory/InputView.java

```diff
@@ -29,7 +30,7 @@
  * @since 0.2
  */
 @PublicEvolving
-public interface InputView {
+public interface InputView extends DataInput {
```

Comment:
Please see the Javadoc, the `InputView` is in little endian, but `DataInput` is in big endian. Making `InputView` extends `DataInput` breaks the API semantics and is error-prone.